ykcompile/src/lib.rs

```diff
@@ -346,16 +302,35 @@ impl<TT> TraceCompiler<TT> {
                             Location::Register(temp)
                         };
                         ty = SIR.ty(&tyid).clone();
+                    } else {
+                        // Dereferencing when the pointee is uncopyable the location to an address.
```

Comment:
Dereferencing a pointer, where the pointee is uncopyable, turns the location to an address?